src/renderer/components/input/input.scss

```diff
@@ -48,7 +48,7 @@
     --flex-gap: #{$padding / 1.5};
 
     position: relative;
-    padding: $padding /4 * 3 0;
+    padding: $padding / 4 * 3 0;
```

Comment:
These are global changes to Input are out-of-scope for this PR. Please describe why these are needed (if they are needed).

Reply:
They are needed to facilitate the usability of `<EditableList>` within `<Wizard>`. I have tried to make sure that there are no visual changes to the rest of the application.